src/AttributesOfNumbers.java

```diff
@@ -0,0 +1,30 @@
+
+import java.util.Scanner;
+import java.util.Random;
+
+public class AttributesOfNumbers {
+    public static void main (String[]args){
+
+        Scanner scan = new Scanner(System.in);
+        Random x = new Random();
+        int luckyNumber;
```

Comment:
ich würd initialisierung deklarierung einer variable nicht auf zwei zeilen splitten - in einer zeile ists kompakter und leichter zu lesen